crates/driver/openapi.yml

```diff
@@ -76,8 +154,107 @@ components:
+          description: |
+            The objective value of the solution.
+
+            `null` indicates that solving was successful but no solution was found.
+          type: number
+          nullable: true
+        signature:
+          description: |
+            Signature confirming that the Solver promised to have this solution for this auction.
+
+            TODO: Decide what is signed and how it is signed.
+          type: string
+    ExecuteRequest:
+      description: Request to the execute endpoint.
+      type: object
+      properties:
+        auctionId:
+          description: Id of the auction that should be executed.
+          type: integer
+    ExecuteResponse:
+      description: Response of the execute endpoint.
+      type: object
+      properties:
+        account:
+          description: The address that executes the transaction.
+          $ref: "#/components/schemas/Address"
+        nonce:
+          description: The nonce of the transaction.
+          type: integer
+        clearingPrices:
+          description: |
+            Mapping of hex token address to price.
+
+            The prices of tokens for settled user orders as passed to the settlement contract.
+          type: object
+          additionalProperties:
+            $ref: "#/components/schemas/BigUint"
+        trades:
+          description: executed orders
+          type: array
+          items:
+            type: object
+            properties:
+              uid:
+                $ref: "#/components/schemas/OrderUID"
+              executedAmount:
+                $ref: "#/components/schemas/BigUint"
+        internalizedInteractions:
+          type: array
+          items:
+            type: object
+            description: |
+              An internalized interaction is an interaction that could have executed on chain but
+              was instead handled through the settlement contract's buffers to save gas. The
+              information included here allows reconstructing the interaction so that it can be
+              checked for validity.
+            properties:
+              calldata:
+                description: hex encoded
+                type: string
+                example: "0x01"
+              inputs:
+                description: Mapping of hex token address to amount.
+                type: object
+                additionalProperties:
+                    $ref: "#/components/schemas/BigUint"
+              outputs:
+                description: Mapping of token address to amount.
+                type: object
+                additionalProperties:
+                    $ref: "#/components/schemas/BigUint"
+        calldata:
+          description: hex encoded
+          type: string
+          example: "0x01"
+        signature:
+          description: |
+            Signature confirming that the Solver promised to execute this auction.
+
+            TODO: Decide what is signed and how it is signed.
```

Comment:
My intuition is that it should be the signature of `nonce || calldata || internalizedInteractions` and it should recover to the `address` in the response. `clearingPrices` are included in the calldata anyway.

OK to leave this "in flux" for now.